test/addons/openssl-providers/binding.cc

```diff
@@ -1,6 +1,9 @@
 #include <assert.h>
 #include <node.h>
+
```

Comment:
```suggestion

#include <openssl/opensslv.h>
```
Otherwise you're implicitly depending on includes from node.h to get OPENSSL_VERSION_MAJOR, which is kind of fragile.

Reply:
Thanks. I was also thinking to include this but since this is part of tests, it would fail once node.h stopped importing openssl headers, if ever. This would be easy to spot then, but being progressive doesn't hurt here.